[2024-11-10T02:20:43.726+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T02:20:42.203017+00:00/task_id=natural_gas_spot_prices_extraction permission to 509
[2024-11-10T02:20:43.726+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T02:20:42.203017+00:00/task_id=natural_gas_spot_prices_extraction permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T02:20:42.203017+00:00/task_id=natural_gas_spot_prices_extraction'
[2024-11-10T02:20:43.754+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T02:20:42.203017+00:00/task_id=natural_gas_spot_prices_extraction permission to 509
[2024-11-10T02:20:43.754+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T02:20:42.203017+00:00/task_id=natural_gas_spot_prices_extraction permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T02:20:42.203017+00:00/task_id=natural_gas_spot_prices_extraction'
[2024-11-10T02:20:43.784+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: natural_gas_spot_prices_etl_pipeline.natural_gas_spot_prices_extraction manual__2024-11-10T02:20:42.203017+00:00 [queued]>
[2024-11-10T02:20:43.791+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: natural_gas_spot_prices_etl_pipeline.natural_gas_spot_prices_extraction manual__2024-11-10T02:20:42.203017+00:00 [queued]>
[2024-11-10T02:20:43.792+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-11-10T02:20:43.804+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): natural_gas_spot_prices_extraction> on 2024-11-10 02:20:42.203017+00:00
[2024-11-10T02:20:43.811+0000] {standard_task_runner.py:57} INFO - Started process 23525 to run task
[2024-11-10T02:20:43.814+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'natural_gas_spot_prices_etl_pipeline', 'natural_gas_spot_prices_extraction', 'manual__2024-11-10T02:20:42.203017+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/eia_natural_gas_spot_prices_etl_pipeline_dag/eia_natural_gas_spot_prices_etl_pipeline_dag.py', '--cfg-path', '/tmp/tmp8jrb9rbk']
[2024-11-10T02:20:43.816+0000] {standard_task_runner.py:85} INFO - Job 35: Subtask natural_gas_spot_prices_extraction
[2024-11-10T02:20:43.858+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T02:20:42.203017+00:00/task_id=natural_gas_spot_prices_extraction permission to 509
[2024-11-10T02:20:43.859+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T02:20:42.203017+00:00/task_id=natural_gas_spot_prices_extraction permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T02:20:42.203017+00:00/task_id=natural_gas_spot_prices_extraction'
[2024-11-10T02:20:43.860+0000] {task_command.py:415} INFO - Running <TaskInstance: natural_gas_spot_prices_etl_pipeline.natural_gas_spot_prices_extraction manual__2024-11-10T02:20:42.203017+00:00 [running]> on host 16801ac1f6e4
[2024-11-10T02:20:43.940+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='natural_gas_spot_prices_etl_pipeline' AIRFLOW_CTX_TASK_ID='natural_gas_spot_prices_extraction' AIRFLOW_CTX_EXECUTION_DATE='2024-11-10T02:20:42.203017+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-10T02:20:42.203017+00:00'
[2024-11-10T02:20:45.137+0000] {logging_mixin.py:151} INFO - Error retrieving metadata: An error occurred (NoSuchKey) when calling the GetObject operation: The specified key does not exist.
[2024-11-10T02:20:48.356+0000] {logging_mixin.py:151} INFO - Error retrieving metadata: An error occurred (NoSuchKey) when calling the GetObject operation: The specified key does not exist.
[2024-11-10T02:20:54.972+0000] {logging_mixin.py:151} INFO - Error retrieving metadata: An error occurred (NoSuchKey) when calling the GetObject operation: The specified key does not exist.
[2024-11-10T02:21:08.680+0000] {logging_mixin.py:151} INFO - Error retrieving metadata: An error occurred (NoSuchKey) when calling the GetObject operation: The specified key does not exist.
[2024-11-10T02:21:09.973+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-11-10T02:21:09.984+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=natural_gas_spot_prices_etl_pipeline, task_id=natural_gas_spot_prices_extraction, execution_date=20241110T022042, start_date=20241110T022043, end_date=20241110T022109
[2024-11-10T02:21:10.008+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-11-10T02:21:10.032+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
